index.rst

```diff
@@ -71,80 +40,85 @@ A general overview of the development flow, as viewed from the user, is as follo
     - Create observing utilities to perform specific tasks, which may be sufficiently generic such that they may be used
```

Comment:
I think that this is where we may want to add a break where the user may want to either continue to flush down the process, creating the utilities themself, or request the new functionality. I know some people will like to continue flushing it down by themself but I think other people will be happy(ier) to deliver the notebook to someone else to finish the job. 

Instead of having the items here, maybe just make these paragraph?

Maybe something like:

> In general, the development workflow, as viewed by the user, starts with a drafting phase. At this point the user utilizes notebooks to flush-out their desired functionality, which may include creating drafts of functions (e.g. for calculating offsets), performing calls to high-level classes (e.g. slew telescope) etc. Once the new functionality has matured enough (which usually means the user ran it on the telescope and is pleased with the results), the user has the option to hand over the notebook to other team members or continue to flush it down by themself. 

Then you would need to rewrite the following paragraphs to match this style... :/

Reply:
I don't think we want to just offer a service like that, we want to encourage people to contribute themselves, not just load it onto the devs. Senior people will ask (and there is some comment lower down saying people should request help if required.